lab-ron/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const gulp = require ('gulp');
+const eslint = require ('gulp-eslint');
+const mocha = require ('gulp-mocha');
+// const watch = require ('gulp-watch');
+
+gulp.task('mocha', function(){
+  return gulp.src('test/*.js').pipe(mocha());
+});
+gulp.task('eslint', function (){
+  return gulp.src(['index.js', 'lib/*.js', 'test/greet-test.js']).pipe(eslint())
```

Comment:
Instead of 'test/greet-test.js' use 'test/*.js', this way your test will automatically add new test files to run and you won't have to add them individually as testing grows.